stdlib/Michelson.ju

```diff
@@ -0,0 +1,58 @@
+mod Michelson =
+
+    -- datatypes
+    type key	 
+    type unit	 
+    type signature	 
+    type chainId	 
+    type option a	 
+    type list a	 
+    type set a	 
+    type operation	 
+    type contract a
+    type pair a b 
+    type either a b	 
+    type lambda a b	 
+    type map a b	 
+    type big-map a b
+
+    -- primitive types
+    type int	 
+    type nat	 
+    type string	 
+    type bytes	 
+    type mutez	 
+    type bool	 
+    type hash-key	 
+    type timestamp	 
+    type address
+
+    -- instructions
+    -- not clear what the types are from documentation (more research needed)
+    -- `words` are typed as `bytes`
+    -- otherwise `a` 
+    sig ext : a -- ???
+    sig dropn : int -> bytes -- return type???
```

Comment:
We won't have these operations (anything which directly manipulates the stack)

Reply:
(there's no stack in the Juvix semantics, we just compile the lambda calculus to it for Michelson)